app/jest.config.js

```diff
@@ -1,13 +1,3 @@
-/** @type {import('jest').Config} */
-const config = {
-  moduleFileExtensions: ['js', 'ts', 'tsx'],
+module.exports = {
   preset: 'react-native',
-  setupFilesAfterEnv: ['@testing-library/jest-native/extend-expect'],
-  testPathIgnorePatterns: ['/node_modules/', '/e2e/'],
-  transformIgnorePatterns: [
-    'node_modules/(?!@react-navigation|@react-native|react-native)',
-  ],
-  collectCoverage: true,
```

Comment:
Why are these configs no longer needed?

Reply:
I should add them back in actually, the upgrade helper added this as a new file and I modified ours to match. Good catch.